DiceRoller/settings.gradle.kts

```diff
@@ -22,12 +24,17 @@ pluginManagement {
 }
 
 dependencyResolutionManagement {
+    @Suppress("UnstableApiUsage")
     repositories {
         google()
         mavenCentral()
     }
 }
 
+plugins {
+    id("org.gradle.toolchains.foojay-resolver-convention") version "0.7.0"
```

Comment:
we should move this to the libs.versions.gradle

Reply:
We can't use version catalogs in settings I think because the settings compile happens before the version catalog compile. :(